test/production/skip-animated-image-import-resize/index.test.ts

```diff
@@ -0,0 +1,20 @@
+import { createNext, FileRef } from 'e2e-utils'
+import path from 'path'
+
+describe('Skip animated image import resize', () => {
+  it('should build with', async () => {
+    const next = await createNext({
+      files: {
+        'pages/index.js': `
+        import Image from 'next/image'
+        import cat from '../public/cat.webp'
+        export default function Home() {
+            return <Image src={cat} alt="cat" />
+        }`,
+        'public/cat.webp': new FileRef(path.join(__dirname, 'cat.webp')),
+      },
+    })
+    expect(true).toBe(true)
```

Comment:
Was this failing to build before? I think a better place for this test is here to verify the behavior actually works:

https://github.com/vercel/next.js/blob/6b49d4744c204d9d572ef2540f092c6195a56bb2/test/integration/image-optimizer/test/util.js#L164-L178

Reply:
Looks like we already have a test for animated webp 🤔 

https://github.com/vercel/next.js/blob/e5be344932589bd71d50b9a08f54f1aeb34df7f8/test/integration/image-optimizer/test/util.js#L212-L226

Reply:
I'll try to refactor these to catch this. :+1: